### PR TITLE
add Elephas to feedstock

### PR DESCRIPTION
**Pull request by Tata17**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/elephas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3
+
+  run:
+    - cython
+    - flask
+    - keras
```
Comment (marcelotrevisani) [inline recipes/elephas/meta.yaml]: According to the ``setup.py``, ``hyperas`` is a dependency os this package
Should we package ``hyperas`` first?

Comment (marcelotrevisani) [reply recipes/elephas/meta.yaml]: I created this PR https://github.com/conda-forge/staged-recipes/pull/9159 and also add you as a maintainer of that recipe.
After we package that project we can continue with this one

Comment (Tata17) [reply recipes/elephas/meta.yaml]: @marcelotrevisani  great, I was just about to push Hyperas myself, thanks!

Comment (Tata17) [reply recipes/elephas/meta.yaml]: @marcelotrevisani and thanks for adding me as a maintainer!

Comment (marcelotrevisani) [reply recipes/elephas/meta.yaml]: Now you can add ``hyperas`` and it should be fine! :)
```suggestion
    - keras
    - hyperas
```

**Pull request merged**